.github/workflows/pr-realm-js.yml

```diff
@@ -92,15 +93,15 @@ jobs:
 
       - name: Restore NPM cache
         id: npm-cache
-        uses: actions/cache@v3
+        uses: actions/cache@v4
         with:
           path: ${{ steps.npm-cache-dir.outputs.dir }}
           key: ${{ runner.os }}-node-${{ hashFiles('package-lock.json') }}
           restore-keys: |
             ${{ runner.os }}-node-
 
       - name: Setup Wireit cache
-        uses: google/wireit@setup-github-actions-caching/v1
+        uses: takameyer/wireit@wireit-caching-gha
```

Comment:
Is this correct? (Same in other places)

Reply:
There is a bit of a thread I started in the private chat about this.  We are getting many warnings about using Node 16, which is coming from different github actions.  This one was quite trivial to fork and change (the original author hasn't touched it in 2 years)

I've cleared up a few of the issues, but there is still some others to take care of (`setup-java`, `setup-android`).  Also I had to keep the `upload-artifacts` to v3 because there's a breaking change in v4 (rabbit hole).

Reply:
Got it, thanks, I've caught up with the thread now 👍